src/main/java/uk/gov/justice/log/utils/PropertyReader.java

```diff
@@ -0,0 +1,180 @@
+package uk.gov.justice.log.utils;
+
+import uk.gov.justice.log.search.SearchCriteria;
+import uk.gov.justice.services.common.converter.ZonedDateTimes;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import net.minidev.json.JSONArray;
+import net.minidev.json.JSONObject;
+import net.minidev.json.parser.JSONParser;
```

Comment:
Why are we using a different new JSON parser for this project? We use Jackson and javax.json classes elsewhere, so why not here?